src/components/m-footer-languages/index.js

```diff
@@ -29,18 +30,40 @@ class AXAFooterLanguages extends BaseComponentGlobal {
     super.init({ styles, template });
   }
 
-  handleClick = (e) => {
-    if (e.target && e.target.dataset && e.target.dataset.language) {
-      e.stopPropagation();
-      this.setAttribute('value', e.target.dataset.language);
+  handleClick = (event) => {
+    const { target } = event;
+    const { lang } = target;
+    const { dataset } = target;
+
+    /**
+     * axa-click event.
+     *
+     * @event FooterLinks#axa-click
+     * @type {object}
+     */
+    const cancelled = fire(this, AXA_EVENTS.AXA_CLICK, lang, {
+      bubbles: true,
+      cancelable: true,
+      composed: true,
+    });
+
+    if (!cancelled) {
+      event.preventDefault();
+    }
+
+    // @todo: we shouldn't need data attributes, since the HTMLElement interface support `lang` DOM Property
```

Comment:
@MarekLacoAXA 
This can be reverted.
The `HTMLElement` interface does define a global `lang` property:)
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/lang